stock_quantity_history_location/models/product.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2019 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import models
+
+
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
+
+    def _get_domain_locations(self):
+        """Ensures the location context isn't overriden"""
+        ctx = dict(self.env.context)
+        if not ctx.get('drop_company_owned') or not ctx.get('company_owned'):
+            return super()._get_domain_locations()
+        ctx.pop('company_owned')
```

Comment:
```suggestion
        ctx.pop('company_owned', None)
```

Reply:
Mmm, what for? I just want to drop it so there's no need of any default value

Reply:
Mmm, what for? I really don't care about the value

Reply:
Sorry, to avoid a `KeyError` if it's missing.

Reply:
Done :)